Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -999,15 +1045,11 @@ theorem Equiv.of_exists {x y : PGame}
   · obtain ⟨j, hj⟩ := hr₁ i
     exact Or.inr ⟨j, Equiv.ge hj⟩
 
-theorem Equiv.of_equiv {x y : PGame} (L : x.LeftMoves ≃ y.LeftMoves)
-    (R : x.RightMoves ≃ y.RightMoves) (hl : ∀ i, x.moveLeft i ≈ y.moveLeft (L i))
-    (hr : ∀ j, x.moveRight j ≈ y.moveRight (R j)) : x ≈ y := by
-  apply Equiv.of_exists <;> intro i
-  exacts [⟨_, hl i⟩, ⟨_, hr i⟩,
-    ⟨_, by simpa using hl (L.symm i)⟩, ⟨_, by simpa using hr (R.symm i)⟩]
-
 @[deprecated (since := "2024-09-26")] alias equiv_of_mk_equiv := Equiv.of_equiv
 
+theorem Equiv.isEmpty (x : PGame) [IsEmpty x.LeftMoves] [IsEmpty x.RightMoves] : x ≈ 0 :=
```

Comment:
This is the only 'major' change of the earlier merge conflict resolution: the newer theorem `birthday_eq_zero` relies on `Equiv.isEmpty` - I've moved it and proved it separately of `Relabelling`.